src/getCoinCombination.test.js

```diff
@@ -7,4 +7,67 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('cents = 0', () => {
+    const actual = getCoinCombination(0);
+    const expected = [0, 0, 0, 0];
+
+    expect(actual).toEqual(expected);
+  });
+
+  it('0 < cents < 5', () => {
+    const actual = getCoinCombination(3);
+    const expected = [3, 0, 0, 0];
+
+    expect(actual).toEqual(expected);
+  });
+
+  it('cents = 5', () => {
```

Comment:
Recall test design techniques and apply these techniques (Equivalence class and boundary value at least)